src/lib/app-logger.js

```diff
@@ -12,7 +12,9 @@ class AppLogger extends Logger {
 			transports: [
 				new (this.deps.winston.transports.Console)({
 					colorize: deps.colorize,
-					level: deps.level
+					level: deps.level,
+					json: this.deps.json,
```

Comment:
__thought:__ there are some interesting (and irrelevant?) inconsistencies here between `deps` and `this.deps` (that existed before your change so probably not worth fixing)